source/users/sessions.txt

```diff
@@ -316,15 +318,35 @@ configure this time for your App's refresh tokens to be anywhere between
 - Custom JWT refresh token expiration is determined by the ``exp`` value
   of the user's JWT.
 
-.. note::
+You can configure the refresh token expiration time for all sessions in
+an App from the Admin UI or Admin API.
+
+Set the Refresh Token Expiration In the Admin UI
```

Comment:
Nit: I don't think we generally refer to an "Admin UI" in our docs? I just did a little clicking around and I see us calling it "App Services UI" in tab sets and other places. I might suggest renaming this either here or elsewhere for consistency.

Reply:
We introduced "The App Services Admin UI" as the full name in our `/app` section IA refresh so it's limited to that section atm. I think if we use it more it has nice parallels since it's basically a UI for the Admin API.